packages/fiori/src/Wizard.ts

```diff
@@ -516,7 +527,7 @@ class Wizard extends UI5Element {
 	_adjustHeaderOverflow() {
 		let counter = 0;
 		let isForward = true;
-		const tabs = this.shadowRoot.querySelectorAll("[ui5-wizard-tab]");
+		const tabs: NodeListOf<Element> = this.shadowRoot!.querySelectorAll("[ui5-wizard-tab]");
```

Comment:
this.shadowRoot!.querySelectorAll<WizardTab>("[ui5-wizard-tab]"); does the same.